Dockerfile

```diff
@@ -8,34 +7,19 @@ WORKDIR /home/app/code
 RUN chown -R app /home/app
 USER app
 
-COPY package.json /home/app/code/package.json
-RUN npm install
-
-COPY Makefile README.md index.js config.js /home/app/code/
-
-RUN mkdir /home/app/code/web
-COPY web/js /home/app/code/web/js
-COPY web/index.html /home/app/code/web/index.html
-COPY web/libs /home/app/code/web/libs
-COPY web/templates /home/app/code/web/templates
+# Copy project's files
+COPY Makefile README.md index.js config.js package.json /home/app/code/
+COPY server/ home/app/code/server
+COPY web/ /home/app/code/web
 
-RUN (cd /home/app/code/web && \
-    ../node_modules/.bin/browserify \
-		-t [ reactify --es6 --target es5 ] \
-		-t brfs \
-		js/entrypoint.js > bundle.js)
-
-COPY web/images /home/app/code/web/images
-COPY web/img /home/app/code/web/img
-COPY web/pics /home/app/code/web/pics
-COPY web/css /home/app/code/web/css
-
-COPY server /home/app/code/server
+# npm install, etc.
+RUN (cd /home/app/code && make)
```

Comment:
Bad approach: `npm install` will have to be re-run each time any file is changed.

Previous code allowed to only run it when package.json was changed.

Secondly, copying `server` after running browserify allows not to re-run browser when unrelated server side code is changed.

Lastly, `make` doesn't need to be installed on production servers... That's why I run browserify directly.